docs/EVM/precompiles/dapp-staking.md

```diff
@@ -1,39 +1,98 @@
-### DappsStaking Precompile
+# DApp Staking Precompile
 
 DappsStaking Precompile enables EVM smart contract to access `pallet-dapps-staking` functionality.
-Example use of this precompile from Contract A:
+Below is the Solidity interface to the precompiles:
 
-```
-import "./DappsStaking.sol";
-contract A {
-    DappsStaking public constant DAPPS_STAKING = DappsStaking(0x0000000000000000000000000000000000005001);
-
-    /// @notice Check current era
-    function checkCurrentEra() public view {
-        uint256 currentEra = DAPPS_STAKING.read_current_era();
-    }
-}
-```
 
-Example use: check `current era` and `total staked amount` in the `pallet-dapps-staking` for Shiden Network. For this example we will use Remix.
+```js
```

Comment:
My personal preference is not to have c/p of code in the docs.

Having a link to the github is sufficient. Duplicating code in the docs doesn't bring any value.